fdbcli/fdbcli.actor.cpp

```diff
@@ -2115,7 +2115,8 @@ ACTOR Future<int> cli(CLIOptions opt, LineNoise* plinenoise, Reference<ClusterCo
 
 				if (tokencmp(tokens[0], "createtenant") || tokencmp(tokens[0], "deletetenant") ||
 				    tokencmp(tokens[0], "listtenants") || tokencmp(tokens[0], "gettenant") ||
-				    tokencmp(tokens[0], "configuretenant") || tokencmp(tokens[0], "renametenant")) {
+				    tokencmp(tokens[0], "configuretenant") || tokencmp(tokens[0], "renametenant") ||
+				    tokencmp(tokens[0], "locktenant") || tokencmp(tokens[0], "unlocktenant")) {
```

Comment:
We could arguably not add these. The prior ones were added because they were included in a release and we wanted to provide compatibility, but the lock ones won't have ever been used.

Reply:
ah ok, I wasn't aware of the history here, will remove them